docs/specification/draft/client/elicitation.mdx

```diff
@@ -151,139 +257,236 @@ To request information from a user, servers send an `elicitation/create` request
 }
 ```
 
-**Reject Response Example:**
+### Out-of-Band Elicitation Requests
+
+Out-of-band elicitation enables servers to direct users to external URLs for interactions that must not pass through the MCP client. This is essential for auth flows, payment processing, and other sensitive or secure operations.
+
+<Note>
+  **Important**: Out-of-band elicitation is *not* for re-authorizing the MCP
+  client's access to the MCP server (that is covered by [MCP
+  authorization](../basic/authorization)). It's specifically for scenarios where
+  the MCP server needs to obtain sensitive information, or authorization for
+  third-party services on behalf of the user. The MCP client's authorization
+  (bearer) token remains unchanged throughout this process.
+</Note>
```

Comment:
```suggestion
  third-party services on behalf of the user. The MCP client's authorization
  (bearer) token remains unchanged throughout this process. Out-of-band
  elicitation is an interaction between the user and the server where the
  client's only role is to facilitate the communication.
</Note>
```
Honestly don't know if this actually makes things better. Just trying to say it in other words for folks who might not get that there are (at least) two distinct auth relationships going on here.

Reply:
Good points. I like "facilitate," but we saw lots of confusion with that with 475. People seemed to think that implied the client needed to do more than was explained in the spec.

I'll rework this a bit to clarify the responsibilities of the client. Thanks for the nudge!